docs/package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "dependencies": {
     "gitbook-cli": "^2.3.0",
+    "gitbook-plugin-anchorjs": "^2.0.1",
```

Comment:
Why is this in this PR?

Reply:
This got added automatically, and I wasn't sure about this so I kept it to get comments